### DISCUSSION TITLE
The page is blank after changing theme Shoka and deploying it.

### DISCUSSION THREAD

**Discussion by Sherry1982** [hexojs/hexo#5170] [Question / Answer]
I deploy page in Gitee, but the balnk page is gotten. The page is normal before changing theme. And when I open index.html, it tells me it can't find app.js and .css files. I look for help from you sincerely.

**Comment by Sherry1982** [comment]
![image](https://user-images.githubusercontent.com/53888121/222937744-84792880-88d4-4c60-bfa5-1f3f4807b934.png)

**Comment by Sherry1982** [comment]
And when it is deployed to Gitee and is opened, ang error doesn't appear...

**Comment by Sherry1982** [comment]
Oh no...When I use previous theme, it can't also work...which indicates the page is blank...

**Comment by Sherry1982** [comment]
OK, it appears, but what should I do to make shoka normal? I really like this theme....

**Comment by stevenjoezhang** [comment]
Can you provide the link of your blog page?

**Comment by Sherry1982** [comment]
Sure, I'm glad to receiver yout e-mail because I even plan to give up...Thank you so much!
The link is:&nbsp;https://jiayzou.gitee.io/zjy_blog/
And the file is provided. I am looking forward to hearing from you.


Sincerely,
Jiayu Zou.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "hexojs/hexo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月9日(星期四) 下午2:32
***@***.***&gt;;
抄送:&nbsp;"$$$$((*邹*&amp;@***@***.******@***.***&gt;;
主题:&nbsp;Re: [hexojs/hexo] The page is blank after changing theme Shoka and deploying it. (Discussion #5170)





 
Can you provide the link of your blog page?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;


从QQ邮箱发来的超大附件

hexoblog.zip (204.38M, 2023年04月08日 14:39 到期)进入下载页面：http://mail.qq.com/cgi-bin/ftnExs_download?t=exs_ftn_download&k=27326365bbc30fc518b9a769153704191e460257030100064b0553040a1a07555f044e500153011b5f0a0500045352535151565d333b365e034a0c075f5851181c5b13650e&code=f2ce3766
